client/js/components/faq/DpFaqList.vue

```diff
@@ -89,11 +89,11 @@ export default {
   },
 
   computed: {
-    ...mapState('faqCategory', {
+    ...mapState('FaqCategory', {
```

Comment:
Is the faq list loading for you?

Reply:
It is. Maybe I fixed it somewhere along. Can't find the commit :-/

Reply:
My fault. Here it is 1e46b284696abc95539062dabfb3a89270ef54c4